readme.md

```diff
@@ -98,9 +98,9 @@ ___Note!___ Project is still under heavy development
 ### v0.1.0
 
 * Basic theme support (c-color, bg-c-color, B-c-color)
-    - New atom and configuration: atoms/_theme-base.scss & $baseColors in _variables.scss
+    - New atom and configuration: atoms/_theme-base.scss & $colors in _variables.scss
```

Comment:
Actually this is editing history of v0.1.0, maybe better to leave these as they were. We just need a new entry for next version stating that this changed.